ByBit.Net/Converters/OrderStopStatusSpotConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Bybit.Net.Enums;
+using CryptoExchange.Net.Converters;
+using System.Collections.Generic;
+
+namespace Bybit.Net.Converters
+{
+    /// <summary>
+    /// Converter for SLTP order statuses
+    /// </summary>
+    internal class OrderStopStatusSpotConverter : BaseConverter<OrderStatus>
```

Comment:
Instead of using this method (which will work fine) it is also possible to use the `EnumConverter` from the CryptoExchange base library. You don't have to write as much boilerplate code as you could just notate the enum values itself with a `MapAttribute` and then set the JsonConverter type to `JsonConverter(typeof(EnumConverter))`

Reply:
Thank you, it looks much comfortable way for mapping.